### PR TITLE
fix deprecation warning

### PR DESCRIPTION
**Pull request by raphael-homann**
### 1. Why is this change necessary?
The message misleads the developper, introducing an unhandled tag

### 2. What does this change do, exactly?
The message leads to the right tag

### 3. Describe each step to reproduce the issue or behaviour.
Keep a payment methos with 'shopware.payment.method.sync' tag, and look at the deprecation log

### 5. Checklist

- [X] I have rebased my changes to remove merge conflicts
- [ ] I have written tests and verified that they fail without my change
- [x] I have created a [changelog file](https://github.com/shopware/platform/blob/trunk/adr/2020-08-03-implement-new-changelog.md) with all necessary information about my changes
- [ ] I have written or adjusted the documentation according to my changes
- [ ] This change has comments for package types, values, functions, and non-obvious lines of code
- [X] I have read the contribution requirements and fulfil them.

### CODE REVIEW

File: changelog/_unreleased/2024-09-10-payment-method-deprecation-log.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: Improve cart page performance
```
Comment (shyim) [inline changelog/_unreleased/2024-09-10-payment-method-deprecation-log.md]: copy paste error?

**Comment by Bird87ZA** [comment]
@raphael-homann please update the changelog. You can use NEXT-00000 as your issue number.

File: changelog/_unreleased/2024-09-10-payment-method-deprecation-log.md (hunk 1 above)
Comment (raphael-homann) [reply changelog/_unreleased/2024-09-10-payment-method-deprecation-log.md]: yes, exact :o

Comment (raphael-homann) [reply changelog/_unreleased/2024-09-10-payment-method-deprecation-log.md]: fixed

**Comment by Bird87ZA** [comment]
Your PR has been merged! Thanks for your contribution 💙

**Pull request closed without merging**